src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -0,0 +1,52 @@
+.pf-c-notification-badge {
+  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-notification-badge--after--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-badge--after--Top: 0;
+  --pf-c-notification-badge--after--Right: 0;
+  --pf-c-notification-badge--after--Width: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--Height: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--BackgroundColor: transparent;
+
+  // This translation works for the Bell icon. If other icons are used, a modifier may need to be created to allow for proper positioning on that icon.
+  --pf-c-notification-badge--after--TranslateX: calc(var(--pf-global--spacer--xs) + var(--pf-c-notification-badge--after--BorderWidth));
```

Comment:
If the goal is to offset the badge by half of it's width, you could use `50%` instead of `var(--pf-global--spacer--xs)`. That's how it looked to me from the designs.

Reply:
I used the spacer plus the border width because @maryshak1996 said above that it was pushed out by 4px. 50% works in this case so I went ahead and changed it in the hope that it might work for other icons without adjustment.